src/containers/Virtual.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
You should rename this file from `Virtual.jsx` to `VirtualEventProject.jsx`. File names should match the name of the component they contain